contracts/mock/Token20.sol

```diff
@@ -0,0 +1,281 @@
+pragma solidity ^0.6.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/Context.sol";
+
+// copied from @openzeppelin/contracts/token/ERC20/ERC20.sol
+contract Token20 is Context, IERC20 {
```

Comment:
you can use the `setupDecimals` method here to https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v3.4.0/contracts/token/ERC20/ERC20.sol#L287
so you can inherit from ERC20 and allow arbitrary number of decimals